### PR TITLE
Add tempo to audio

### PR DESCRIPTION
**Pull request by tkell**
This change is [Reviewable](https://reviewable.io/reviews/algorithmic-music-exploration/amen/91)

### CODE REVIEW

**Comment by tkell** [comment]
Paging @bmcfee:  I added tempo, and added a `track` level Timing, so we can get mean tempo for an entire track easily.  What do you think?

File: amen/audio.py [hunk 1]
```diff
@@ -250,6 +252,26 @@ def _get_chroma(self):
...
+        Gets tempo data from librosa, and returns it as a feature collection.
+
+        Parameters
+        ---------
+
+        Returns
+        -----
+        FeatureCollection
+        """
+        centroids = librosa.feature.spectral_centroid(self.analysis_samples)
```
Comment (bmcfee) [inline amen/audio.py]: this doesn't get used, right?

**Comment by bmcfee** [comment]
> Paging @bmcfee: I added tempo, and added a track level Timing, so we can get mean tempo for an entire track easily. What do you think?

Cool, thanks for adding that.  I'm not sure mean tempo is the right aggregation; if it's half in 120 and half in 160, i don't think 140 is a reasonable summary.  Median or mode might be a better aggregator.. what do you think?

File: amen/audio.py (hunk 1 above)
Comment (tkell) [inline amen/audio.py]: Wups, stray copy/paste.  Ty!

**Comment by tkell** [comment]
Yeah, I'll go for median - will merge later tonight.

**Pull request merged**